### DISCUSSION TITLE
Better orders

### DISCUSSION THREAD

**Discussion by hnappinn** [FakeFishGames/Barotrauma#11908] [Ideas and suggestions]
### Issue: 
The game has an uncomfortable ordering system. You have to change them often and the limit is only 3.
### Solution option:
Add several sets of orders that can be changed. It would be handy to add this via the button 
![image](https://user-images.githubusercontent.com/59388392/234646144-14d951e3-c365-46e5-b010-7b43a1738c40.png)

**Comment by MarkCringan** [comment]
I would love an order option for "Refuel Reactor" so it can be a priority 1 and doesn't glue somebody to the reactor like "Operate Reactor".
